.github/workflows/wheel-builder.yml

```diff
@@ -118,16 +114,17 @@ jobs:
           - {ARCH: 'x86', WINDOWS: 'win32'}
           - {ARCH: 'x64', WINDOWS: 'win64'}
         PYTHON:
-          - {VERSION: "2.7", TOXENV: "py27", MSVC_VERSION: "2010"}
-          - {VERSION: "3.5", TOXENV: "py35", MSVC_VERSION: "2019"}
-          - {VERSION: "3.6", TOXENV: "py36", MSVC_VERSION: "2019"}
-          - {VERSION: "3.7", TOXENV: "py37", MSVC_VERSION: "2019"}
-          - {VERSION: "3.8", TOXENV: "py38", MSVC_VERSION: "2019"}
-    name: "Python ${{ matrix.PYTHON.VERSION }} on ${{ matrix.WINDOWS.WINDOWS }}"
+          - {VERSION: "2.7", MSVC_VERSION: "2010"}
+          - {VERSION: "3.5", MSVC_VERSION: "2019"}
+          - {VERSION: "3.6", MSVC_VERSION: "2019"}
+          - {VERSION: "3.7", MSVC_VERSION: "2019"}
+          - {VERSION: "3.8", MSVC_VERSION: "2019"}
+          - {VERSION: "3.8", MSVC_VERSION: "2019", "USE_ABI3": "true", "ABI_VERSION": "cp36"}
```

Comment:
I win the fight on bcrypt but lose it here?

Reply:
You lose it here for now. I don't want to have our wheel infra only capable of building the new style when we don't know the results of that experiment. If bcrypt goes well we can update this before 3.1 release.

Reply:
I have confidence in the bcrypt expirement, so I guess I'm ok waiting a month to delete it :-)